src/ganache-server.ts

```diff
@@ -9,25 +9,19 @@ function send(msg: Message): void {
   process.send(msg);
 }
 
-function setupServer({ accountsConfig, gasLimit, gasPrice, coverage }: Options) {
-  const ganacheOpts = {
-    accounts: accountsConfig,
-    gasLimit,
-    gasPrice: `0x${gasPrice.toString(16)}`,
-  };
-
-  if (!coverage) {
-    return ganache.server(ganacheOpts);
+function setupServer(nodeOptions: NodeOptions): any {
+  if (!nodeOptions.coverage) {
+    return ganache.server(nodeOptions);
   } else {
     return require('ganache-core-coverage').server({
-      ...ganacheOpts,
+      ...nodeOptions,
       emitFreeLogs: true,
       allowUnlimitedContractSize: true,
```

Comment:
I think this belongs in `getConfig`. There is already a conditional that modifies Ganache options for coverage runs.

Reply:
True. Keep in mind next PR is `solidity-coverage` and this code will be removed.